scripts/Samuel-Ajayi.js

```diff
@@ -1,4 +1,3 @@
 function	hng(name, id,language, email) {
-	console.log(`Hello world, this is ${name} with HNGi7 ID ${id} and ${email} using ${language} for stage 2 task`)
-}
-hng ("Ajayi Samuel Toluwa", "HNG-00302", "javascript", "troysammie7@gmail.com")
+	console.log(`Hello world, this is ${name} with HNGi7 ID ${id} ${email} using ${language} for stage 2 task`)
```

Comment:
console.log(`Hello World, this is ${name} with HNGi7 ID ${id} and email ${email} using ${language} for stage 2 task`)
update yours to look like this one.